obspy/io/sac/sactrace.py

```diff
@@ -459,10 +459,10 @@ def _strgetter(hdr):
     def get_str(self):
         try:
             # value is a bytes
-            value = self._hs[HD.STRHDRS.index(hdr)].decode()
+            value = unicode(self._hs[HD.STRHDRS.index(hdr)].decode())
         except AttributeError:
             # value is a str
-            value = self._hs[HD.STRHDRS.index(hdr)]
+            value = unicode(self._hs[HD.STRHDRS.index(hdr)])
```

Comment:
Is there any chance that SAC has unicode values? Otherwise maybe just use `str()` to do the conversion? The `future` library currently translates that to ` unicode()` call on Python 2 but we hope to get rid of that eventually.